src/Nethermind/Nethermind.Blockchain/SynchronizationManager.cs

```diff
@@ -213,8 +214,12 @@ public void HintBlock(Keccak hash, UInt256 number, NodeId receivedFrom)
 
         public void AddNewTransaction(Transaction transaction, NodeId receivedFrom)
         {
-            if (_logger.IsTrace) _logger.Trace($"Received a pending transaction {transaction.Hash} from {receivedFrom}");
-//            _transactionStore.AddPending(transaction, HeadNumber);
+            if (_logger.IsTrace)
+            {
```

Comment:
_logger calls always in one line - every other 'if' with brackets

Reply:
Ok, will update.